CHANGES

```diff
@@ -36,6 +36,9 @@ Features added
   Patch by Rouslan Korneychuk.
 * 10938: doctest: Add :confval:`doctest_show_successes` option.
   Patch by Trey Hunner.
+* #10678: Emit "source-read" events for files read via
```

Comment:
Let's order it by # (so I assume that it should come before 10938). At least, it will save some work for @AA-Turner when doing the next release

Reply:
I tend to order it chronologically, by which change was introduced first -- but I'm inconsistent, and I do sort by other methods sometimes.

A